New page Integrations

ckarmy opened the pull request.

ckarmy commented:
Sorry for the new PR but I don`t know how to force it

kusma commented on integrations.md:
```diff
@@ -0,0 +1,86 @@
+permalink: /integrations/
+redirect_from:
+  - /source-code/
+  - /downloads/
```
These redirects are wrong. You don't want to take over these urls...

ckarmy replied:
And I delete the redirects? Or which are?

kusma replied:
Well, you don't want the URLs https://fuseopen.com/source-code/ or https://fuseopen.com/downloads/ to redirect to https://fuseopen.com/integrations/, I assume? I guess you copied get_started.md, which has these there to keep old links working. Just delete the "redirect_from"-list, and it should be fine.

ckarmy replied:
Yes I delete it on the new commit

kusma commented on integrations.md:
```diff
@@ -0,0 +1,86 @@
+---
+
+
+<a name="packages"></a>
```
`<a name="packages"></a>`... what's this supposed to do?

ckarmy replied:
Nothing sorry

kusma replied:
Ah, it creates linkable anchors for the list. But I guess it should wrap the headings then, shouldn't it?

ckarmy replied:
I delete it on the new commit

kusma commented on integrations.md:
```diff
@@ -0,0 +1,86 @@
+### Packages/Samples
+- <a href="https://github.com/rbtech/fuse-lottie" target="_blank"> Lottie animations: integration of Lottie animations on Fuse</a>
+- <a href="https://github.com/naosk8/fuse-homography-image-uploader" target="_blank"> Homography image transformation: A sample project for Fuse, an image uploader supporting Homography image transformation.</a>
+- <a href="https://github.com/jeroensmeets/mastodon-app" target="_blank"> 11t: 11t is an iOS and Android app for connecting to Mastodon</a>
```
I think this would be better as either a data-file (see the _data folder), or simply as a few lists in the front-matter that gets looped over...

kusma replied:
By the way, I think just adding it as frontmatter is both the simplest and cleanest solution.

kusma replied:
Done this here:
https://github.com/kusma/fuse-open.github.io/tree/integrations-page

But thinking about it a little bit more, perhaps it would be even cleaner if this was done using collections? That way we could use markdown in the description as well, and avoid putting HTML in the yml... Oh well, that can be done later, I suppose.

kusma commented on integrations.md:
You left this one here also...

ckarmy replied:
Sorry. Delete it on the new commit

kusma commented:
I rewrote your list as frontmatter yml, and using liquid to generate the html, and I think the result is much nicer. I also made the contents a bit more structured and squashed the history while at it.

You can see the result here:
https://github.com/kusma/fuse-open.github.io/tree/integrations-page

@ckarmy: What do you think? If you agree, you can adopt that version instead...

kusma commented on _config.yml:
```diff
@@ -20,6 +20,7 @@ header_pages:
   - url: /examples/
     title: Examples
   - showcases.md
+  - integrations.md
```
Hmm, this makes the list a bit too long so it wraps before it disappears into the hamburger menu...

I guess the alternatives is:
1. Use a shorter name
2. Use a smaller font
3. Change the breakpoints so it collapse to the hamburger menu earlier
4. Let it wrap

I'm not sure what alternative I think is the best...

ckarmy replied:
I take the number 2!

kusma replied:
Sounds great!

ckarmy commented:
> I rewrote your list as frontmatter yml, and using liquid to generate the html, and I think [the result](https://github.com/kusma/fuse-open.github.io/commit/5453f6016db9caa76820fd9da160e08cfbb640eb#diff-6aaecd52b77ec090161d765dc473267a) is much nicer IMO. I also made the contents a bit more structured and squashed the history while at it.
> 
> You can see the result here:
> https://github.com/kusma/fuse-open.github.io/tree/integrations-page
> 
> @ckarmy: What do you think? If you agree, you can adopt that version instead...

Yes much better! Thanks! Is in the new commint!

ckarmy commented:
all ready? Or do I need to change something else?

kusma commented:
@ckarmy: Great work, thanks!
